plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -105,6 +105,7 @@
 import io.trino.spi.type.TimestampType;
 import io.trino.spi.type.TimestampWithTimeZoneType;
 import io.trino.spi.type.TypeManager;
+import io.trino.spi.type.VarcharType;
```

Comment:
Please update `iceberg.md`.

Reply:
Updated with docs on how to time travel to branches/tags. Let me know how that looks!